### PR TITLE
Cleanups: drop use of % in logging

### PR DESCRIPTION
**Pull request by dwaynebailey**
logging can accept variables for interpolation as parameters.  This
means they're only interpolated if something is logged.  Using % means
that the interpolation is always happening (as far as I understand).

### CODE REVIEW

**Comment by phlax** [comment]
hmm - my feeling is stick to interpolation its more pythonic and it works everywhere

**Comment by dwaynebailey** [comment]
Perhaps some other motivators:

- Pythonic for %, maybe but consider that:
  - There's also ``.format()``
  - It's also the [proposed model of how to do this](https://docs.python.org/2/library/logging.html#logging.Logger.debug)
- We're inconsistent and use both for no other reason then I think we forgot that logging doesn't need %.
- And perhaps these [better motivations](https://www.simonmweber.com/2014/11/24/python-logging-traps.html) which make things more reliable it would seem.

File: pootle/apps/accounts/utils.py [hunk 1]
```diff
@@ -287,16 +287,14 @@ def revert_units_reviewed(self):
                 previous_review = reviews.latest('pk')
                 unit.reviewed_by = previous_review.submitter
                 unit.reviewed_on = previous_review.creation_time
-                logger.debug("Unit reviewed_by reverted: %s"
-                             % (repr(unit)))
+                logger.debug("Unit reviewed_by reverted: %s", (repr(unit)))
```
Comment (unho) [inline pootle/apps/accounts/utils.py]: Unnecessary parentheses around `repr`.

File: pootle/apps/accounts/utils.py [hunk 2]
```diff
@@ -287,16 +287,14 @@ def revert_units_reviewed(self):
...
             else:
                 unit.reviewed_by = None
                 unit.reviewed_on = None
 
                 # Increment revision
                 unit._target_updated = True
-                logger.debug("Unit reviewed_by removed: %s"
-                             % (repr(unit)))
+                logger.debug("Unit reviewed_by removed: %s", (repr(unit)))
```
Comment (unho) [inline pootle/apps/accounts/utils.py]: Unnecessary parentheses around `repr`.

File: pootle/apps/accounts/utils.py [hunk 3]
```diff
@@ -334,8 +332,7 @@ def revert_units_state_changed(self):
                 # Increment revision
                 unit._state_updated = True
                 unit.save()
-                logger.debug("Unit state reverted: %s"
-                             % (repr(unit)))
+                logger.debug("Unit state reverted: %s", (repr(unit)))
```
Comment (unho) [inline pootle/apps/accounts/utils.py]: Unnecessary parentheses around `repr`.

File: pootle/apps/pootle_app/management/commands/update_stores.py [hunk 4]
```diff
@@ -56,8 +56,8 @@ def handle_store(self, store, **options):
         disk_mtime = store.get_file_mtime()
         if not options["force"] and disk_mtime == store.file_mtime:
             # The file on disk wasn't changed since the last sync
-            logging.debug(u"File didn't change since last sync, skipping "
-                          u"%s" % store.pootle_path)
+            logging.debug(u"File didn't change since last sync, "
+                          "skipping %s", store.pootle_path)
```
Comment (unho) [inline pootle/apps/pootle_app/management/commands/update_stores.py]: String should start with `u"`.

File: pootle/apps/pootle_store/models.py [hunk 5]
```diff
@@ -1849,9 +1849,9 @@ def update_from_disk(self, overwrite=False):
                 )
             self.last_sync_revision = update_revision
             if update_unsynced:
-                logging.info(u"[update] unsynced %d units in %s [revision: %d]"
-                             % (update_unsynced, self.pootle_path,
-                                update_revision))
+                logging.info(u"[update] unsynced %d units in %s "
+                             "[revision: %d]", update_unsynced,
```
Comment (unho) [inline pootle/apps/pootle_store/models.py]: String should start with `u"`.

File: pootle/apps/pootle_app/management/commands/update_stores.py (hunk 4 above)
Comment (dwaynebailey) [inline pootle/apps/pootle_app/management/commands/update_stores.py]: Not in a line continuation I believe.

**Comment by unho** [comment]
Nits aside, lgtm.

File: pootle/apps/pootle_app/management/commands/update_stores.py (hunk 4 above)
Comment (unho) [inline pootle/apps/pootle_app/management/commands/update_stores.py]: Perhaps it is not a requirement (I thought it was but couldn't find anything supporting this idea) but IMHO it make easier to spot this is the continuation of the previous line unicode string.

**Pull request merged**